### PR TITLE
DM-44243: Add worker pods configuration on rubintv-dev summit deployment

### PR DESCRIPTION
**Pull request by sebastian-aranda**

### CODE REVIEW

**Review by mareuter — APPROVED** [review]
This looks fine. One suggestion to look at.

File: charts/rubintv/templates/vault-secrets.yaml [hunk 1]
```diff
@@ -7,3 +7,21 @@ metadata:
...
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: butler-secret
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  path: "{{ .Values.global.vaultSecretsPath }}/butler-secret"
+  type: Opaque
+---
```
[the remark this answers is not in the document]
Comment (sebastian-aranda) [reply charts/rubintv/templates/vault-secrets.yaml:20]: that's correct, however we are using the pull-secret for rapid-analysis images, which come from the nexus repository.

[the remark this answers is not in the document]
Comment (sebastian-aranda) [reply charts/rubintv/templates/vault-secrets.yaml:20]: I just got it now jeje 😅, I'll remove it from the deployment-frontend.yml 👍.

[the remark this answers is not in the document]
Comment (sebastian-aranda) [reply charts/rubintv/templates/vault-secrets.yaml:20]: so I took a closer look and the pull secrets are defined at the root of the values.yaml. This is defined for the rubintv shared chart and we will need it for the production app too (not yet, but at some point we will do), so I would prefer to keep it as it is defined.

**Pull request merged**